pkg/pipeline/write/grpc/server.go

```diff
@@ -0,0 +1,77 @@
+package grpc
```

Comment:
looks like this grpc server is only used for test, so it could be moved under the test package? Or is it meant to be imported from other projects?

Reply:
Indeed I'm reusing this into CLI:
https://github.com/netobserv/network-observability-cli/blob/c7615024c987813d3368921f99c598edafd4feb8/cmd/flow-capture.go#L89-L94

We do the same in FLP from eBPF agent gRPC definition:
https://github.com/netobserv/flowlogs-pipeline/blob/df307e0ef280d54ca73d19407809be14652be146/pkg/pipeline/ingest/ingest_grpc.go#L48-L49